yaus_c4gt_frontend/src/pages/Home.js

```diff
@@ -0,0 +1,630 @@
+          Show Statistics
+        </Button>
+      ),
+    },
+    {
+      date: "2022-07-19",
+      name: "Sunbird",
+      url: <a href=" https://yaus.xyz/sunbird371"> https://yaus.xyz/sunbird371</a>,
+      views: "68",
+      install: "-",
+      clicks: "125",
+      open: "110",
+      action: (
+        <Button onClick={() => setVisible(true)} type="primary">
+          Show Statistics
+        </Button>
+      ),
+    },
+    {
+      date: "2022-07-19",
+      name: "Sunbird",
+      url: <a href="https://yaus.xyz/sunbird437">https://yaus.xyz/sunbird437</a>,
+      views: "56",
+      install: "-",
+      clicks: "124",
+      open: "149",
+      action: (
+        <Button onClick={() => setVisible(true)} type="primary">
+          Show Statistics
+        </Button>
+      ),
+    },
+    {
+      date: "2022-07-19",
+      name: "Sunbird",
+      url: <a href="https://yaus.xyz/sunbird752">https://yaus.xyz/sunbird752</a>,
+      views: "96",
+      install: "-",
+      clicks: "134",
+      open: "131",
+      action: (
+        <Button onClick={() => setVisible(true)} type="primary">
+          Show Statistics
+        </Button>
+      ),
+    },
+    {
+      date: "2022-07-03",
+      name: "Competency Passbook",
+      url: <a href="https://yaus.xyz/passbook">https://yaus.xyz/create</a>,
+      views: "68",
+      install: "-",
+      clicks: "168",
+      open: "128",
+      action: (
+        <Button onClick={() => setVisible(true)} type="primary">
+          Show Statistics
+        </Button>
+      ),
+    },
+  ];
+
+
+
+  const suffix = (
+    <AudioOutlined
+      style={{
+        fontSize: 16,
+        color: "white",
+      }}
+    />
+  );
+  const onSearch = (value) => console.log(value);
+
+  const onChange1 = (pagination, filters, sorter, extra,data) => {
+    console.log("params", pagination, filters, sorter, extra,data);
+  };
+
+
+  return (
+    <>
+      <div class="MyApp">
+
+        <Space direction="vertical" size={12}>
+          <RangePicker />
+        </Space>
+
+
+        <Modal
+          title="Statistics"
+          centered
+          visible={visible}
+          onOk={() => setVisible(false)}
+          onCancel={() => setVisible(false)}
+          width={1200}
+
+        >
+          <>
+          <div> <Card bordered={true}><h3> https://yaus.xyz/education </h3></Card></div>
+            <Row className="rowgap-vbox" gutter={[24, 0]}>
+              {count.map((c, index) => (
+                <Col
+                  key={index}
+                  xs={24}
+                  sm={24}
+                  md={12}
+                  lg={6}
+                  xl={6}
+                  className="mb-24"
+                >
+                  <Card bordered={false} className="criclebox ">
+                    <div className="number">
+                      <Row align="middle" gutter={[24, 0]}>
+                        <Col xs={18}>
+                          <span>{c.today}</span>
```

Comment:
Please add question mark in object properties

`c?.today`

Reply:
resolved in 20af2631700c22af6c6c3a27edef4477140e9d34